src/logout/components/Logout/Logout.js

```diff
@@ -3,10 +3,12 @@ import { func } from 'prop-types';
 
 export default class Logout extends React.Component {
   static propTypes = {
-    logout: func.isRequired
+    logout: func.isRequired,
+    emptyAllRequests: func.isRequired
```

Comment:
I'd prefer if we continue to only pass a single `logout` function prop in here, and extend what the function does within `logout/components/Logout/index.js`.